src/makeCalculator.js

```diff
@@ -37,7 +37,37 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    operate(callback, x) {
+      callback.call(this, x);
+
+      return this;
+    },
+
+    add(x) {
```

Comment:
```suggestion
    add(x = 0) {
```
lets add default values to make it functional without argument